lib/airbrake/rack/notice_builder.rb

```diff
@@ -45,6 +54,9 @@ def add_context(notice)
 
         context[:url] = @request.url
         context[:userAgent] = @request.user_agent
+        context[:httpMethod] = @request.request_method
+        context[:referer] = @request.referer
+        context[:headers] = request_headers
```

Comment:
Airbrake API is free to throw away such properties, because they are not officially supported.